xdp-filter/python-tests/xdp-filter-exec.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cd ..
+./xdp-filter $@
```

Comment:
Why is this needed? When run from the test harness, $XDP_FILTER should always be set, no?

Reply:
I used this since I was running the tests from the test folder and so xdp-filter could not find object files for the XDP programs. I will rewrite it so that the working directory is the the one containing the compiled files and remove this file. `XDP_FILTER` variable actually does not get inherited into the Python, since it is just local, will fix that.

Reply:
Setting the work directory to where the binary is doesn't work after install (we don't want to run with workdir in /usr/bin). Just exporting XDP_FILTER so the Python code can get it from the environment should be enough, no?